backend/app/adapter/sqldb/migration/19_add_constraints_to_user_changelog_table.sql

```diff
@@ -12,8 +10,7 @@ ALTER TABLE "user_changelog"
 ALTER TABLE "user_changelog"
     DROP CONSTRAINT "user_changelog_pkey";
 ALTER TABLE "user_changelog"
-    ADD COLUMN email CHARACTER VARYING(254) NOT NULL;
-ALTER TABLE "user_changelog"
-    ADD CONSTRAINT "user_changelog_pkey" PRIMARY KEY (email);
```

Comment:
Should the email not be a primary key?

Reply:
I see it's because the user id should be the primary key correct?

Reply:
@oatovar It would be primary key. We are facing some issues here. @kapillamba4 can explain the details.

Reply:
https://github.com/short-d/short/issues/824

Reply:
Thank you for creating the issue @kapillamba4 to track the necessary changes.